docs/docs/guides/labs/components/using-environment-variables-in-components.md

```diff
@@ -0,0 +1,89 @@
+---
+title: 'Using environment variables with components'
+sidebar_position: 100
+---
+
+
+import Preview from '@site/docs/partials/\_Preview.md';
+
+<Preview />
+
+
+It's common to want to configure components depending on the environment in which they are run. `dg` and components make it easy to configure this behavior.
```

Comment:
```suggestion
With `dg` and components, you can easily configure components depending on the environment in which they are run. 
```

Reply:
I would give a brief description of the example here: "To demonstrate, we'll walk through setting up an example ELT pipeline with a Sling component that reads Snowflake credentials from environment variables"